set.mm

```diff
@@ -674802,6 +674806,831 @@ fixed reference functional determined by this vector (corresponding to
+      ACUWGVWAVUSWQYPXDWRYKSUWGVVMUFUVTUWKUGUWGVEZVVMVEZUWKVEUVFWLZXEVVLVUJVVRA
+      VUJVVKXOVVLCUWGVUIVULVUGVVKAVUGVUJVUSPPZWQXDVVLVVKVVSVULVVKVQVVLCUWGVVFWX
+      BWQXDUWGVVMUVTVUIVVFWWSWWTUVGRVVLCUWGVVNWXBWQTVVLVVGVVNCVULVVKUDUEUXCVVMA
+      WWMVUJWWNPUVHYCTYKYKWUSTZSAVUHVVDVUQWURSSUXGCBUXEUYRVYBUVIWLZPZUYFVUFYEVV
+      AUXPUWJCUXOUWIBUVTEUWCUYGVURUWIVEZYQZWDQAWWOUWDWWRPYOTUWFUYAUWRUWFUXKUXQU
+      YAUYHYRUWFUXTUWQUBURURUWFURYNUWFUWMURLZMZUXSUWOUWPWXIUXRUWNUXEUWKWXIVUTWX
+      HUWTUTUXRUWNKZWXIVUTWXHUWTUWFVUTWXHWXEPUWFWXHVQUWFUWTWXHVUFPYECUXOUWIBUVT
+      UWMUWCUYGVURWXFYQZQAWWPUWDWXHWWQXEYOYPUVJXDYEAUWEUWSYTUWDAUVTUWIEUWCUBAUD
+      UECUXCUVTVUSWWNWXCAVUJVVKUTUYCVUKVWKVVGVVFVUIUXCJKAVUJUYCVVKGUVKAVUJVVKVU
+      KVWMYSAVUJVVKVWKVWPYSUXGUXCBVUIVVFUYRVUCXRRUVLZUYEWXFVFPTXAXBAUAUWAUVSAUW
+      EUWDAUWEMZUWDUYBWXMUXKUXQUYAWXMUWHUXKWXMUWHUWLUWRAUWEUWSWXMUWEUWSWXMUVTUW
+      IEUWCUBAUVTVCLZUWEWXLPAUYDUWEUYEPZWXFVFVGVHZVNZWXMUWHUXKAUWHUXKUQZUWEAUWT
+      UXKUQWXRAUWTUXKACUXGUWCVUQUVMXAAUWTUWHUXKACUWGUWCVUSWQZYPXDPWRUVNWXMUXPUW
+      JUWKUXEWXMVUTUYDUWTUXPUWJKAVUTUWEWXDPZWXOWXMUWTUWHWXQAUWTUWHYTUWEWXSPTZWX
+      GRWXMUWHUWLUWRWXPWFAWWOUWEWWRPZUVOWXMUWRUYAWXMUWHUWLUWRWXPYRWXMUWQUXTUBUR
+      WXMWXHMZUWOUXSUWPWYCUWNUXRUWKUXEWYCUXRUWNWYCVUTWXHUWTWXJWXMVUTWXHWXTPWXMW
+      XHVQWXMUWTWXHWYAPWXKRWDWXMWWOWXHWYBPYOYPUVPXDYEWXMBUXOEUWCUBAUYCUWEGPWXOU
+      YGVFTXAXBUVQAVVQWXNMUWBAVVQWXNWXAWXLSUVTUVRWLS $.
+  $}
+
+  ${
+    $d K j $.  $d R a i j $.  $d U j $.  $d j ph $.
+    primrootsunit.1 $e |- ( ph -> R e. CMnd ) $.
+    primrootsunit.2 $e |- ( ph -> K e. NN ) $.
+    primrootsunit.3 $e |- U = { a e. ( Base ` R ) |
+     E. i e. ( Base ` R ) ( i ( +g ` R ) a ) = ( 0g ` R ) } $.
+    $( Primitive roots are have left inverses.  (Contributed by metakunt,
+       25-Apr-2025.) $)
+    primrootsunit $p |- ( ph -> ( ( R PrimRoots K ) =
+     ( ( R |`s U ) PrimRoots K ) /\ ( R |`s U ) e. Abel ) ) $=
+      ( vj cv cplusg cfv co c0g wceq cbs wrex crab nfv cbvrexw primrootsunit1
+      oveq1 eqeq1d rabbii eqtri ) ABCJEFGHCDKZFKZBLMZNZBOMZPZDBQMZRZFUMSJKZUHUI
+      NZUKPZJUMRZFUMSIUNURFUMULUQDJUMULJTUQDTUGUOPUJUPUKUGUOUHUIUCUDUAUEUFUB $.
+  $}
+
+  ${
+    $d E l x y $.  $d K l x y $.  $d M l x y $.  $d R a c i $.  $d R l x y $.
+    $d U c $.  $d U l x y $.  $d c i ph $.  $d l ph x y $.
+    primrootsCoprimePow.1 $e |- ( ph -> R e. CMnd ) $.
+    primrootsCoprimePow.2 $e |- ( ph -> K e. NN ) $.
+    primrootsCoprimePow.3 $e |- ( ph -> E e. NN ) $.
+    primrootsCoprimePow.4 $e |- ( ph -> ( E gcd K ) = 1 ) $.
+    primrootsCoprimePow.5 $e |- ( ph -> M e. ( R PrimRoots K ) ) $.
+    primrootsCoprimePow.6 $e |- U = { a e. ( Base ` R ) |
+     E. i e. ( Base ` R ) ( i ( +g ` R ) a ) = ( 0g ` R ) } $.
+    $( Coprime powers of primitive roots are primitive roots.  (Contributed by
+       metakunt, 25-Apr-2025.) $)
+    primrootsCoprimePow $p |- ( ph -> ( E ( .g ` R ) M ) e.
```

Comment:
label capitalization lol (Coprime -> [coprm](https://us.metamath.org/mpeuni/coprmgcdb.html))

Incidentally it looks like these are coprime multiples